app/requirements.txt

```diff
@@ -4,6 +4,7 @@ django-filter==0.11.0
 django-extensions==1.6.1
 django-cors-headers==1.0.0
 google-api-python-client==1.5.0
+oauth2client==2.0.1
```

Comment:
Without this line I was getting import errors from https://github.com/WorldBank-Transport/DRIVER/blob/develop/app/driver_auth/views.py#L10 when I tried to log in to the admin site, even after reprovisioning a couple times; but aside from that I'm not sure if it needs to be there, there may be a better way to solve the issue.